schema/viewer.js

```diff
@@ -28,13 +28,17 @@
  * additional fields below.
  */
 
+import CausalityJWT from './causality_jwt';
+import Me from './me';
 import Sales from './sales';
 import { GraphQLObjectType } from 'graphql';
 
 const ViewerType = new GraphQLObjectType({
   name: 'Viewer',
   description: 'A Viewer',
   fields: () => ({
+    causality_jwt: CausalityJWT,
```

Comment:
Yeah, when I dove in I was really surprised - two hard constraints on two super super common use-cases, and not well explained in the docs.